Auth/AuthContextProvider.tsx

```diff
@@ -13,35 +20,37 @@ type AuthProviderProps = {
 
 export const AuthContext = createContext<AuthContextType | undefined>(undefined);
 
-function AuthContextProvider({children}: AuthProviderProps) {
+function AuthContextProvider({ children }: AuthProviderProps) {
+  const { environment } = useAppEnvironment();
 
-  const {environment, setEnvironment} = useAppEnvironment();
-
-  async function loggedIn() {
-    const valid = await authTokenIsValid(environment);
-    if (valid) {
-      setAuthenticationState(AuthenticationState.AUTHENTICATED, null);
-      return true;
-    }
-
-    const refreshSuccess = await refreshAuth(environment);
-    return refreshSuccess;
-  }
-
-  const [authenticated, setAuthenticated] = useState<AuthenticationState | null>(() => getAuthenticationState());
+  const [authenticated, setAuthenticated] = useState<AuthenticationState | null>(() =>
+    getAuthenticationState(),
+  );
 
   useEffect(() => {
+    async function loggedIn() {
+      const valid = await authTokenIsValid(environment);
+      if (valid) {
```

Comment:
As a general stylistic note, I would say you can remove any variable that's defined and then only used once. (2 instances in this function, but please check/fix elsewhere as well)